sources/web/datalab/polymer/modules/github-file-manager/github-file-manager.ts

```diff
@@ -60,6 +60,8 @@ interface GhFileResponse {
  */
 class GithubFileManager extends BaseFileManager {
 
+  cache = new GithubCache();
```

Comment:
Shouldn't this be a static instance? I'm not sure when we'd use multiple `GithubFileManager` instances, but it's probably not a bad idea to reuse the same cache.

Reply:
I'd say we don't need a separate file for just the cache object, can we inline it here as a static property?

Reply:
Done. I had originally expected there would be more to it.